### PR TITLE
Display Stats When Rendering Pokémon

### PR DESCRIPTION
**Pull request by Tomm1128**
When rendering the pokemon to the screen, whether on page load or search, the Pokémon's hp, weight, height and types are displayed as well. 

Screenshot 2024-05-28 at 3 26 40 PM

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -6,26 +6,58 @@ const handleUserInput = (event) => {
   getPokemon(userInput)
 }
 
+const getPokemonTypes = (pokemon) => {
+  const pokemonTypes = pokemon.types
+  let typeArray = []
+  pokemonTypes.forEach((types) => {
+    typeArray.push(types.type.name)
+  })
+  return typeArray
```
Comment (stephenmckeon) [inline index.js]: Smells like `.map()`

**Review by stephenmckeon — APPROVED** [review]

**Comment by Tomm1128** [comment]
@stephenmckeon - I forgot to add the units of measure to the height and weight. I just did so. I also implemented the map if you need to do another review.

File: index.js [hunk 2]
```diff
@@ -1,31 +1,61 @@
...
   event.preventDefault()
   const userInput = event.target[0].value
   getPokemon(userInput)
 }
 
+const getPokemonTypes = (pokemon) => {
+  const pokemonTypes = pokemon.types
+  let typeArray = []
+  pokemonTypes.map(type => typeArray.push(type.type.name))
+  return typeArray
```
Comment (stephenmckeon) [inline index.js]: Try this:
```suggestion
  return pokemonTypes.map(type => type.type.name)
```

**Comment by Tomm1128** [comment]
Ready for another review @stephenmckeon

**Review by stephenmckeon — APPROVED** [review]

**Pull request merged**